pkg/pods/pod.go

```diff
@@ -551,6 +551,7 @@ func (pod *Pod) getLaunchable(launchableStanza LaunchableStanza, runAsUser strin
 			Fetcher:          uri.DefaultFetcher,
 			RootDir:          launchableRootDir,
 			P2exec:           pod.P2exec,
+			ExecNoLimit:      true,
```

Comment:
i figured this isn't worth the trouble to bubble up to preparer config but LMK if you disagree